AMW_angular/io/src/app/shared/elements/loading-indicator.component.scss

```diff
@@ -15,7 +15,7 @@
   display: inline-block;
   top: 50%;
   transform: translateY(-50%);
-  background-image: url('src/assets/images/ajax-loader.gif');
```

Comment:
The loader doesn't show up with the new path (404 on https://${host}/assets/images/ajax-loader.gif)
With the old path I get this error:
```
[INFO] 12 08 2020 17:42:32.661:INFO [launcher]: Starting browser ChromeHeadless
[ERROR]
[ERROR] ERROR in ./src/app/shared/elements/loading-indicator.component.scss
[ERROR] Module Error (from ./node_modules/postcss-loader/src/index.js):
[ERROR] (Emitted value instead of an instance of Error) CssSyntaxError: liima\AMW_angular\io\loading-indicator.component.scss:18:2: Can't resolve './src/assets/images/ajax-loader.gif' in 'liima\AMW_angular\io\src\app\shared\elements'
[ERROR]
[ERROR]   16 |   top: 50%;
[ERROR]   17 |   transform: translateY(-50%);
[ERROR] > 18 |   background-image: url('src/assets/images/ajax-loader.gif');
[ERROR]      |  ^
[ERROR]   19 |   height: 32px;
[ERROR]   20 |   width: 32px;
```
@mburri you have an idea? I tried also with `~/assets/images/ajax-loader.gif` and `assets/images/ajax-loader.gif` but it didn't work.

Reply:
ok, not pretty but relative path from the scss works...